index.html

```diff
@@ -17,11 +18,17 @@ <h1 class="welcome-title"> Welcome to Real Dev Squad </h1>
     <h2 class="info-note highlighted-text">You can check out all our current events happening <a href="./events.html">here</a></h2>
     <img class="logo-img" src="img/Real-Dev-Squad@1x.png" alt="Real Dev Squad logo" />
     <h3 class="info-note"> To know how this page is getting developed, join our Discord Channel. Contact one of the existing members </h3>
+    <div class="cta-button__container " id = "buttonDiv">
+      <h4 align="center">Please subscribe for further event updates</h4>
+    </div>
   </main>
 
   <footer>
     <p class="info-repo">The contents of this website are deployed from this <a href="https://github.com/Real-Dev-Squad/website" target="_blank" rel="noopener noreferrer">open sourced repo</a></p>
   </footer>
-
+  <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.3/umd/popper.min.js" integrity="sha384-vFJXuSJphROIrBnz7yo7oB41mKfc8JzQZiCq4NCceLEaO4IHwicKwpJf9c9IpFgh" crossorigin="anonymous"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0-beta.2/js/bootstrap.min.js" integrity="sha384-alpBpkh1PFOepccYVYDB4do5UnbKysX5WZXm3XxPqe5iKTfUKjNkCk9SaVuEZflJ" crossorigin="anonymous"></script>
```

Comment:
@ankushdharkar just checking if we're using jquery and I'm not aware about that discussion